packages/jest-serializer/src/__tests__/index.test.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```

Comment:
Some Flow maybe?

Reply:
Ah, pardon this is a test (so it's not necessary to flow it, but I like to mix these)